SpringContainerMS/chart/springcontainerms/templates/deployment.yaml

```diff
@@ -59,6 +64,12 @@ spec:
             value: "{{ .Values.kafka.rejectedOrdersTopicName }}"
           - name: KCSOLUTION_ALLOCATED_ORDERS
             value: "{{ .Values.kafka.allocatedOrdersTopicName }}"
+          - name: LOGGING_CONTAINER_MS_ROOT
+            value: "{{ .Chart.logging.root }}"
```

Comment:
Should this be `.Values` and not `.Chart`?

Reply:
Indeed